src/test/java/uk/gov/pay/directdebit/gatewayaccounts/resources/GatewayAccountResourceIT.java

```diff
@@ -305,37 +291,17 @@ public void shouldCreateAGatewayAccountWithMinimumDetails() throws JsonProcessin
                 .header("Location", is(documentLocation))
                 .body("gateway_account_id", is(notNullValue()))
                 .body("gateway_account_external_id", startsWith("DIRECT_DEBIT:"))
-                .body("service_name", is(SERVICE_NAME))
                 .body("payment_provider", is(PAYMENT_PROVIDER.toString()))
                 .body("type", is(TYPE.toString()))
                 .body("description", is(nullValue()))
                 .body("analytics_id", is(nullValue()));
     }
 
-    @Test
-    public void shouldReturnBadRequestIfValidationFails() throws JsonProcessingException {
-        String postBody = new ObjectMapper().writeValueAsString(ImmutableMap.builder()
-                .put(PAYMENT_PROVIDER_KEY, PAYMENT_PROVIDER.toString())
-                .put(TYPE_KEY, TYPE.toString())
-                .put(DESCRIPTION_KEY, DESCRIPTION)
-                .put(ANALYTICS_ID_KEY, ANALYTICS_ID)
-                .build());
-
-        givenSetup()
-                .body(postBody)
-                .post(GATEWAY_ACCOUNTS_API_PATH)
-                .then()
-                .contentType(JSON)
-                .statusCode(422)
-                .body("errors[0]", is("serviceName may not be empty"));
-    }
```

Comment:
Rather than being deleted, I think this test might want to be replaced by something that checks what happens when validation fails for another field (payment provider or something).

Reply:
Good shout. I've amended the test to validate the description length.